chapters/1.md

```diff
@@ -1,100 +1,185 @@
 
-* Fungibility: Because of the on-by-default nature of the privacy technologies, Monero is fungible, meaning one Monero will always be equal to another. This means you will not need to worry about Monero being blacklisted by exchanges or vendors.
+Monero’s privacy features allow the network to assess the validity of a transaction and determine whether or not the sender has a sufficient account balance, without the actually knowing the transaction amount or account balances! Nobody can view others’ account balances, and transactions don’t reveal the source of the funds being transferred. 
 
-## Monero Goal
+These are marked with “???” in the diagram, since an no outside observer can ascertain the values. The mechanics behind these unique privacy features are discussed in chapter X.
 
-The Monero Mission is providing privacy by default. No strange setup, no settings, Monero provides you unique techniques by default for protecting you and your funds. The Monero Community and The Monero Core Team want to create a powerful and helpful system in order to help you in protecting your privacy by reading this book and resources (official websites, books).
+[Figure X.3]
+## Principles of Monero
 
-## Why open source?
+Monero is designed with the following principles in mind:
 
-Satoshi Nakimoto created the blockchain with a goal to have people place trust in each other rather than in centralized third parties. Open-source software supports this goal, as anyone can review the code to make sure that it is secure. For example, the Linux kernel is open-source and has over 5,062 contributors, and it is widely considered to be a highly secure operating system.
+* **Network Decentralization**: The Monero network and ledger are globally distributed. This means that there is no single server or database that can be maliciously controlled or censored. If one government were to shut down Monero nodes in their country, or attempt to limit who can send/receive Monero, the effort would be in vain! The rest of the world will carry on the network and process any valid transactions.
 
-The benefits of open-source software, namely trust and security, align well with the goals of cryptocurrency, which is why the code to The Monero Project is completely open-source.
+* Financial Security: The globally-distributed Monero has no central weak point that can be hacked to steal your funds. Your Moneroj are secured by incorrigible cryptographic techniques, so there is no need to trust a third party with responsibility over your funds. Every single Monero participant can verify the validity of the ledger themselves, so you do not even have to trust the node operators! (You can learn more about the cryptographic techniques that secure Monero in chapter X)
```

Comment:
While literally "incorrigible" means something that cannot be changed, in English it carries connotations of a stubborn person, as in "My mother in law is an incorrigible person." Possibly use `immutable` or `indelible`.